src/Microsoft.Azure.WebJobs.Host/packages.config

```diff
@@ -1,17 +1,67 @@
+  <package id="System.IO.FileSystem" version="4.3.0" targetFramework="net46" />
+  <package id="System.IO.FileSystem.Primitives" version="4.3.0" targetFramework="net46" />
+  <package id="System.Linq" version="4.3.0" targetFramework="net46" />
+  <package id="System.Linq.Expressions" version="4.3.0" targetFramework="net46" />
   <package id="System.Linq.Queryable" version="4.3.0" targetFramework="net45" />
+  <package id="System.Net.Http" version="4.3.1" targetFramework="net46" />
+  <package id="System.Net.Primitives" version="4.3.0" targetFramework="net46" />
   <package id="System.Net.Requests" version="4.3.0" targetFramework="net45" />
+  <package id="System.Net.Sockets" version="4.3.0" targetFramework="net46" />
+  <package id="System.ObjectModel" version="4.3.0" targetFramework="net46" />
+  <package id="System.Reflection" version="4.3.0" targetFramework="net46" />
+  <package id="System.Reflection.Extensions" version="4.3.0" targetFramework="net46" />
+  <package id="System.Reflection.Primitives" version="4.3.0" targetFramework="net46" />
+  <package id="System.Resources.ResourceManager" version="4.3.0" targetFramework="net46" />
+  <package id="System.Runtime" version="4.3.0" targetFramework="net46" />
+  <package id="System.Runtime.Extensions" version="4.3.0" targetFramework="net46" />
+  <package id="System.Runtime.Handles" version="4.3.0" targetFramework="net46" />
+  <package id="System.Runtime.InteropServices" version="4.3.0" targetFramework="net46" />
+  <package id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" targetFramework="net46" />
+  <package id="System.Runtime.Numerics" version="4.3.0" targetFramework="net46" />
+  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net46" />
+  <package id="System.Security.Cryptography.Encoding" version="4.3.0" targetFramework="net46" />
+  <package id="System.Security.Cryptography.Primitives" version="4.3.0" targetFramework="net46" />
+  <package id="System.Security.Cryptography.X509Certificates" version="4.3.0" targetFramework="net46" />
   <package id="System.Spatial" version="5.8.2" targetFramework="net45" />
+  <package id="System.Text.Encoding" version="4.3.0" targetFramework="net46" />
+  <package id="System.Text.Encoding.Extensions" version="4.3.0" targetFramework="net46" />
+  <package id="System.Text.RegularExpressions" version="4.3.0" targetFramework="net46" />
+  <package id="System.Threading" version="4.3.0" targetFramework="net46" />
+  <package id="System.Threading.Tasks" version="4.3.0" targetFramework="net46" />
+  <package id="System.Threading.Timer" version="4.3.0" targetFramework="net46" />
+  <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="net46" />
+  <package id="System.Xml.XDocument" version="4.3.0" targetFramework="net46" />
   <package id="WindowsAzure.Storage" version="7.2.1" targetFramework="net45" />
```

Comment:
When you upgrade, Nuget checks to see if they need to be updated (which is only required if the package has a net46 version). Otherwise, you can leave them alone -- changing to net46 would still bring in the same (highest available) package.